src/test/java/uk/gov/mca/beacons/api/WebMvcTestConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.mca.beacons.api;
+
+import org.springframework.context.annotation.Bean;
+import uk.gov.mca.beacons.api.services.GetValidationErrorResponse;
+import uk.gov.mca.beacons.api.services.GetValidationErrorResponseService;
+
+public class WebMvcTestConfiguration {
```

Comment:
Imported in the unit tests for the controllers to ensure that an instance of `GetValidationErrorResponse` is defined within the Spring context.